lib/vc.js

```diff
@@ -334,29 +324,18 @@ function _getId(obj) {
  * @private
  */
 function _checkPresentation(presentation) {
-  // check context cardinality
-  if(jsonld.getValues(presentation, '@context').length < 2) {
-    throw new Error(
-      '"@context" property needs to be an array of two or more contexts.');
-  }
-
   // ensure first context is 'https://www.w3.org/2018/credentials/v1'
   if(presentation['@context'][0] !== 'https://www.w3.org/2018/credentials/v1') {
     throw new Error(
       'https://www.w3.org/2018/credentials/v1 needs to be first in the ' +
       'list of contexts.');
   }
 
-  // check type presence and cardinality
-  if(!presentation['type']) {
-    throw new Error('"type" property is required.');
-  }
+  const types = jsonld.getValues(presentation, 'type');
 
-  if(!Array.isArray(presentation['type']) ||
-    presentation['type'].length < 2 ||
-    !presentation['type'].includes('VerifiablePresentation')) {
-    throw new Error(
-      '"type" must be `VerifiablePresentation` plus specific type.');
+  // check type presence
+  if(!presentation['type'] || !types.includes('VerifiablePresentation')) {
```

Comment:
```suggestion
  if(!(presentation.type && types.includes('VerifiablePresentation'))) {
```

Note that the above is still not right. It should be checking if `type` is a string or array and normalizing and comparing or something along those lines.